src/common/components/vanilla-modules/button/customButton.css

```diff
@@ -5,14 +5,8 @@
 .hasSpinner .text {
   /* hide text without chaning button width */
   opacity: 0;
-}
-
-.spinner {
```

Comment:
All these styles (and additional 'spinner' wrapper) are needed so that button doesn't change the width when spinner appears.